BUILDING.md

```diff
@@ -44,7 +44,7 @@ Trace Compass can be traced by doing the following in the launch configuration:
 * -Djava.util.logging.config.file=%gitroot%/logging.properties (where %gitroot% is the directory tracecompass is checked out to)
 * -Dorg.eclipse.tracecompass.logging=true
```

Comment:
Should we add some description here on how to use the other handlers and how to make it work with the `-Xbootclasspath` setting? We should also have a reference link to the trace-event-logger documentation for more information.

The README.md in this repo should mention about tracing Trace Compass and have link to the chapter in BUILDING.md.